jobs/build/tarball-sources/Jenkinsfile

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def release = load("pipeline-scripts/release.groovy")
+    def buildlib = release.buildlib
+    def commonlib = release.commonlib
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'COMPONENTS',
+                        description: 'Brew component name or NVR',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "logging-fluentd-container"
+                    ],
+                    [
+                        name: 'ADVISORY',
+                        description: 'Image release advisory number. If not given, the number will be retrived from ocp-build-data.',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'OUT_DIR',
+                        description: 'Output dir of elliott tarball-sources',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "/mnt/nfs/home/jenkins/container-sources/"
+                    ],
+                    [
+                        name: 'RCM_GUEST',
+                        description: 'Details of RCM GUEST',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/rcm-guest/puddles/RHAOS/container-sources/"
+                    ],
+                    [
+                        name: 'MAIL_LIST_SUCCESS',
+                        description: 'Success Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'release-engineering@redhat.com',
```

Comment:
This email address is apparently no longer valid:
```
SSUE: Your email request to release-engineering@redhat.com was not
received because the email address as a contact point to Release
Configuration Management team (RCM) has been decommissioned.
```

Reply:
what has been the reason this comment has been marked as resolved?

Reply:
I don't think anyone needs a success email... engineer who runs it will know it ran, anyone else can check the build log. How about just remove the success email?

Reply:
`MAIL_LIST_SUCCESS` isn't even used below. just nuke it.